guides/common/modules/proc_reclaiming-space-from-on-demand-repositories.adoc

```diff
@@ -0,0 +1,26 @@
+[id="reclaiming-space-from-on-demand-repositories_{context}"]
+= Reclaiming Space From On Demand Repositories
+
+When the download policy is set to on demand, the {Project} downloads packages only when the clients request them.
+You can clean up these packages to reclaim space.
+
+.For a single repository
+
```

Comment:
```suggestion
```

You don't need an empty live below headings starting with a dot. Please fix all instances.